streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentTestUtils.java

```diff
@@ -112,6 +113,56 @@ public final class AssignmentTestUtils {
 
     private AssignmentTestUtils() {}
 
+    public static AssignmentConfigs getConfigsWithLagWithStandby(final long acceptableRecoveryLagMs) {
```

Comment:
There were a HUGE number of individual calls to the `AssignmentConfigs` constructor in the assignor-related tests, which means we'd have to update a ridiculous number of lines just to add a single config to this simple container class. There are better ways to do this right but I tried to help the situation without completely blowing up the scope, so that's what this is